app/src/main/java/com/gpayinterns/chat/LoginActivity.java

```diff
@@ -199,10 +208,13 @@ public void onResponse(JSONObject response)
                             String message = response.getString("message");
                             if(message.equals("Success"))
```

Comment:
You can make this condition `active && message.equals("Success")` instead of the nesting.